README.md

```diff
@@ -34,6 +52,51 @@ It is a standalone Microservice that connects the two platforms and provides a s
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Supported payment methods
+
+The PAYONE Integration Service currently supports the following payment methods:
+
+* Direct debit
+  Available in: worldwide
+* Advance payment  
+  Available in: Europa
+  
+* [Creditcards](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+3-D+Secure)
+  Available in: worldwide
+    
+  * Visa  
+  * MasterCard   
+  * American Express  
+  * JCB   
+  * Diners Club  
+      
+* E-Wallets
+  
+  * [Paypal](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+PayPal) 
+    Available in: over 200 countries 
+  * [paydirekt](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+paydirekt)  
+    Available in Germany
+  
+* Online transfer  
+  * Sofortbanking
+    Available in: Austria, Belgium, Germany, The Netherlands, Italy, Poland, Spain, Switzerland, United Kingdom
+  * giropay
+    Available in: Germany
+  * eps (electronic payment standards)  
+    Available in: Austria
+  * [PostFinance E-Finance](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+Postfinance)  
+    Available in: Suisse
+  * [PostFinance Card](https://docs.payone.com/display/public/PLATFORM/Special+remarks+-+Postfinance)  
+    Available in: Suisse
+  * [iDEAL](https://docs.payone.com/display/public/PLATFORM/Special+Remarks+-+Recurring+Transactions+with+iDeal)  
+    Available in: Netherlands
+  * Bancontact  
+    Available in: Belgium 
+      
+* Financing  
+  * [Klarna](https://github.com/commercetools/commercetools-payone-integration/blob/master/docs/Klarna-Activation.md)  
```

Comment:
Would be nice to link to our payment specific integration docs also for other methods.

Reply:
we only have a internal documentation for klarna :-)....I have double checked it

Reply:
Wouldn't it be nice to have integration docs for all methods? WDYT @butenkor

Reply:
A lot payment-methods has the same workflow and for each "special" step in the workflow, I have linked the related payone-documentation, so I think that is enought